Dockerfile

```diff
@@ -163,5 +170,51 @@ RUN yum erase -y cscope && \
     for p in ../patches/*.patch; do patch -p1 < "$p"; done && \
     ./configure --prefix=/usr && \
     make -j$(nproc) && \
-    make install && \
-    cd /tmp && rm -r cscope
+    make install DESTDIR=/tmp/cscope_install
+
+FROM main as sudoers
+COPY add_user_to_sudoers /tmp/add_user_to_sudoers
+RUN cd /tmp/add_user_to_sudoers && \
+    cargo build --release && \
+    mkdir -p /tmp/sudoers_install && \
+    install -m 4755 -t /tmp/sudoers_install target/release/add_user_to_sudoers
```

Comment:
Did you want to clean this up?

Reply:
why would I? I am totally abandoning this image except for the install dir

Reply:
OK, nifty